### DISCUSSION TITLE
Schema reloading support?

### DISCUSSION THREAD

**Discussion by AttilaOrs** [prest/prest#573] [API Server] [answered]
Hi all, I am evaluating pREST, and i am not sure how schema reloading is handled. 
I am thinking something like [this]( https://postgrest.org/en/v7.0.0/admin.html?highlight=schema%20change#schema-reloading)

**Comment by avelino** [comment, marked as answer]
What would be schema reloading ?

The pREST reads the query return and executes it in json (or xml) at runtime, since you are doing a query on a table, if it is a view it is what is structured in the view
